### PR TITLE
Add new workflow to update homebrew formula on release

### PR DESCRIPTION
**Pull request by cjappl**
This workflow was found here:

https://github.com/dawidd6/action-homebrew-bump-formula

This should be very similar to our AUR workflow, which can be seen here:
https://github.com/wfxr/forgit/blob/ab7400ddb37c758b8247ce0884cd2283a873e258/.github/workflows/aur.yaml#L1-L20

I am writing an email to @wfxr asking him to add a BREW_TOKEN to our secrets with the appropriate permissions so this can run :). This change should not be submitted until he has confirmed the BREW_TOKEN has been added to our secrets.

### CODE REVIEW

File: .github/workflows/brew.yaml [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+on:
+  workflow_run:
+    workflows:
+      - "Release"
+    types:
+      - completed
+
+jobs:
+  aur:
```
Comment (carlfriedrich) [inline .github/workflows/brew.yaml:11]: This job should be called "brew" or something similar.

File: .github/workflows/brew.yaml [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+    runs-on: ubuntu-latest
+    steps:
+      - name: Update Homebrew formula
+        uses: dawidd6/action-homebrew-bump-formula@v3
+        with:
+          # Required, custom GitHub access token with the 'public_repo' and 'workflow' scopes
+          token: ${{secrets.TOKEN}}
```
Comment (carlfriedrich) [inline .github/workflows/brew.yaml:18]: And the token should probably be called `HOMEBREW_TOKEN` to make its purpose clear.

**Review by carlfriedrich — CHANGES_REQUESTED** [review]
@cjappl Great stuff, thanks for checking this out!

**Review by carlfriedrich — APPROVED** [review]

**Comment by carlfriedrich** [comment]
@cjappl Thanks for the quick fixes.
Just being curious: did you generate a personal access token in your account for this purpose, so that the automatic brew commits will be done in your name?

**Comment by cjappl** [comment]
> @cjappl Thanks for the quick fixes. Just being curious: did you generate a personal access token in your account for this purpose, so that the automatic brew commits will be done in your name?

No, I could, but it's probably safer from a security side to have wfxr do it (just so I don't have to send one of my tokens in plaintext to him via email or similar). He is still the one that must update any secrets or other Settings of this project, as it's under his name.

I would expect the commits likely end up under his user account in that case!

**Review by wfxr — APPROVED** [review]
LGTM.

@cjappl BREW_TOKEN has been set.

**Comment by cjappl** [comment]
Thanks @wfxr ! Will merge sometime tomorrow.

**Comment by carlfriedrich** [comment]
@cjappl Did you merge into `FixZshCompletions` on purpose?

**Comment by cjappl** [comment]
Whoa. No I did not. Thank you for catching..... Let me fix that

**Comment by cjappl** [comment]
Great catch @carlfriedrich . would you please review #342 and #340 to make sure they are both as you expect them to be?

**Pull request merged**